tap_hibob/streams.py

```diff
@@ -10,22 +11,42 @@
 # TODO: Delete this is if not using json files for schema definition
 SCHEMAS_DIR = Path(__file__).parent / Path("./schemas")
 
-from tap_hibob.schemas import Employees
+from tap_hibob.schemas import Employees, EmployeeHistory
+
 
 class EmployeesStream(HibobStream):
     """Define custom stream."""
+
     name = "employees"
     path = "/v1/people"
     primary_keys = ["id"]
     records_jsonpath = "$.employees[*]"
     replication_key = "creationDateTime"
     schema = Employees.schema
 
-    def get_url_params(self, context: Optional[dict], next_page_token: Optional[Any]) -> Dict[str, Any]:
+    def get_url_params(
+        self, context: Optional[dict], next_page_token: Optional[Any]
+    ) -> Dict[str, Any]:
         params = super().get_url_params(context, next_page_token)
         params["showInactive"] = "true"
         params["includeHumanReadable"] = "true"
         return params
 
-
-
+    def get_child_context(self, record: dict, context: Optional[dict]) -> dict:
+        """Return a context dictionary for child streams."""
+        return {
+            "employee_id": record["id"],
+        }
+
+
+class EmployeeHistoryStream(HibobStream):
+    _LOG_REQUEST_METRIC_URLS = True
+    name = "employee_history"
+    path = "/v1/people/{employee_id}/employment"
+    primary_keys = ["id", "employee_id"]
+    records_jsonpath = "$.values[*]"
+    replication_method = "INCREMENTAL"
+    replication_key = "effectiveDate"
+    is_timestamp_replication_key = True
```

Comment:
Oh cool! What does this do?

Reply:
We'll have to make sure this works loading into stitch, because it can be quite picky about timestamps